resources/views/print/send.blade.php

```diff
@@ -0,0 +1,15 @@
+<div class="card">
+    <div class="card-header">@lang('print.transfer_money')</div>
+    <div class="card-body">
+        <form method="POST" action="{{ route('print.send') }}">
+            @csrf
+            <select class="selectpicker" data-live-search="true" id="receiver_id" name="receiver_id" title="@lang('print.user')">
```

Comment:
Just a note: we have a search blade that could be included here after #152 is resolved.